### DISCUSSION TITLE
Fixing replication by volume.move not by volume.fix.replication

### DISCUSSION THREAD

**Discussion by amir-latifi** [seaweedfs/seaweedfs#5925] [Q&A]
Hi, 
I want to change replica placement of some volumes from 001 to 010. But for some reasons i don't want to and can't do it by volume.fix.replication. Instead, i do these steps for this change:
1. Change volume replica placement from 001 to 010 by volume.configure.replication
2. Move the volume to a volumeserver on a different rack with volume.move

Does this scenario work fine? Am I missing something?

**Comment by Material-Scientist** [comment]
I think you have to use `fs.configure -replication=010 -apply` in weed shell for that.

**Reply by amir-latifi** [reply]
No. I think there is no need for --apply
